docs/user/tutorials/install_epinio_on_eks.md

```diff
@@ -0,0 +1,94 @@
+#  AWS EKS configuration
+
+## Create an EKS cluster
+
+If you don't have an existing cluster, follow the [quickstart](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html) to create an EKS cluster.
+
+## EKS Prerequisites
+
+* We follow the documented recommendation of having at least 2 nodes for EKS
+* Epinio v0.0.21 has been tested with AWS EKS incl. kubernetes version v1.20.4
+* To do more extensive testing we recommend a 2 node cluster with "t3.xlarge" instances
+* To just try out Epinio, e.g. 2 "t3a.large" are sufficient
```

Comment:
```suggestion
* To just try out Epinio, just 2 "t3a.large" instances are sufficient
```

Reply:
I'd leave it like that (the "e.g."). There are nearly 400 instance types within AWS, and I choosed "t3a.large" for it's price (2VCPUs, 8GB RAM, 5GBE) - but there are similar, and even cheaper ones the user could join for other reasons ...

Reply:
The problem I have with the `e.g` is syntactical. The sentence  makes no proper sense to me with it. How about
```
To just try out Epinio just 2 "t3a.large" (or similar) instances are sufficient
```
I added the `(or similar)`.